docs/spec/class-compat.rst

```diff
@@ -89,6 +89,10 @@ annotated in ``__init__`` or other methods, rather than in the class::
       def __init__(self, content):
           self.content: T = content
 
+``ClassVar`` cannot be used as a qualifier for a :ref:`TypedDict <typeddict>`
+item or a :ref:`NamedTuple <namedtuple>` field. Such usages are also hard
```

Comment:
I don't think the term "hard error" is used anywhere else in the spec. I recommend changing to "Such usage also generates an error at runtime".

(Same comment for `Final` below.)